Packs/MicrosoftDefenderAdvancedThreatProtection/TestPlaybooks/playbook-Microsoft_Defender_Advanced_Threat_Protection-Test.yml

```diff
@@ -1645,7 +1648,7 @@ tasks:
     conditions:
     - label: "yes"
       condition:
-      - - operator: isNotEmpty
+      - - operator: isEmpty
```

Comment:
This is not the same condition, was this intentional?

Reply:
Since I had to replace the previous software with a new one, I couldn't find any software with vulnerabilities so I had to change it to isEmpty.. I can change it to just take the first element from a list but then then we wouldn't be able to predict the tasks afterwards (which conditions to put)